.gitignore

```diff
@@ -1,3 +1,4 @@
 *.pyc
 *.swp
+.coverage*
```

Comment:
We don't want to ignore changes to .coveragerc

Reply:
Got it, will update.